components/PostFooterBox.tsx

```diff
@@ -94,15 +94,18 @@ const PostFooterBox: React.ElementType = () => {
       >
         While you're here...
       </h3>
-      <p>
+      <p
+        style={{
+          lineHeight: 1.75,
+        }}
+      >
         We're a student-run organization committed to providing hands-on
         experience in journalism, digital media and business for the next
         generation of reporters. Your support makes a difference in helping give
         staff members from all backgrounds the opportunity to develop important
         professional skills and conduct meaningful reporting. All contributions
         are tax-deductible.
       </p>
-
       <DonationForm />
       <br></br>
       <TextButtonWithLink
```

Comment:
Do we want the changes in this file?

Reply:
Yes, I cleaned up the design a tiny bit. Removed the red line at the top since I thought it was ugly and changed the spacing to make it a bit more compact.